.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
 tests/output
+.vscode/settings.json
```

Comment:
Can we keep this out of the gitignore as we try to keep IDE specific things out of `.gitignore`.

Reply:
made change